doc/dev/custom_devices.rst

```diff
@@ -77,14 +77,14 @@ Additionally, all measurements will always return ``true``.
 
 In addition to implementing the ``QuantumDevice`` class, one must implement an entry point for the
 device library with the name ``<DeviceIdentifier>Factory``, where ``DeviceIdentifier`` is used to
-uniquely identify the entry point symbol. As an example, we use the identifier ``CustomDevice``:
+uniquely identify the entry point symbol.
+You can use the `GENERATE_DEVICE_ENTRYPOINT(DEVICE_NAME, NAMESPACE)` template macro to define this entry
+point where `DEVICE_NAME` is the name of the device identifier and `NAMESPACE` is the device class namespace
+in the C++ format. As an example, we use the identifier ``CustomDevice``:
```

Comment:
Not sure, hopefully this would be clear to an external implementer 🤔 
```suggestion
For simplicity, you can use the ``GENERATE_DEVICE_ENTRYPOINT(DEVICE_NAME, NAMESPACE)`` macro to
define this function, where ``DEVICE_NAME`` is the name of the C++ device class, and ``NAMESPACE``
is the C++ namespace of the class. Note that in this case the ``<DeviceIdentifier>`` symbol has to
match the device class name, as ``DEVICE_NAME`` will be used to refer to both in the macro.
As an example, we use the identifier ``CustomDevice`` for the class defined above:
```